README.md

```diff
@@ -14,32 +14,35 @@ If you want to get data from the chain you tend to have to use a provider like i
 
 If you wish to build a big data lake or even just fetch dynamic events from the chain this task is near impossible without a third party paid tool or something like thegraph hoster services and most do not let you pull in millions of rows at a time fast. This data should be able to be fetched for free, blazing fast and customisable to your needs. This is what reth-indexer does.
 
-This project aims to solve this by reading directly from the reth node db and indexing the data into a postgres database you can then query fast with indexes already applied for you. You can also scale this easily by running multiple instances of this program on different boxes and pointing them at the same postgres database (we should build that in the tool directly).
+This project aims to solve this by reading directly from the reth node db and indexing the data into traditional databases that you can then query fast with indexes already applied for you. You can also scale this easily by running multiple instances of this program on different boxes and pointing them at the same underlying database (we should build that in the tool directly).
 
 This tool is perfect for all kinds of people from developers, to data anaylsis, to ML developers to anyone who wants to just get a snapshot of event data and use it in a production app or just for their own reports.
 
 ## Features
 
-- Creates postgres tables for you automatically
+- Creates database tables for you automatically (postgres, gcp biquery, or others)
 - Creates indexes on the tables for you automatically to allow you to query the data fast
+- Can write index to multiple data stores (i.e. postgres, gcp bigquery, etc)
 - Indexes any events from the reth node db
 - Supports indexing any events from multiple contracts or all contracts at the same time
 - Supports filtering even on the single input types so allowing you to filter on every element of the event
 - Snapshot between from and to block numbers
 - No code required it is all driven by a json config file that is easy to edit and understand
-- Created on your own infurstructure so you can scale it as you wish
-- Exposes a ready to go API for you to query the data
+- Created on your own infrastructure so you can scale it as you wish
+- Exposes a ready to go API for you to query the data (postgres implementation only at this time)
```

Comment:
ah yeah makes sense here as you wouldnt want to query bigquery anyway like this as it would get super expensive with an API direct calls